tests/trace_plugin_test.py

```diff
@@ -102,6 +103,19 @@ def test_TraceApi(self) :
                 break
         self.assertTrue(isTrxInBlockFromTraceApi)
 
+        # relaunch with no time allocated for http response & abi-serializer. Will fail because get_info fails.
+        node.kill(signal.SIGTERM)
+        Utils.Print("Ignore expected: ERROR: Node relaunch Failed")
+        isRelaunchSuccess = node.relaunch(timeout=10, addSwapFlags={"--http-max-response-time-ms": "0", "--abi-serializer-max-time-ms": "10"})
+
+        cmdDesc="get block_trace"
+        cmd=" --print-response %s %d" % (cmdDesc, blockNum)
+        cmd="%s %s %s" % (Utils.EosClientPath, node.eosClientArgs(), cmd)
```

Comment:
Why do it in 2 steps, and not:
` cmd="%s %s --print-response %s %d" % (Utils.EosClientPath, node.eosClientArgs(), cmdDesc, blockNum)`

Reply:
No reason except that it was copied from another location that builds it up similar.